src/ansys/fluent/core/codegen/datamodelgen.py

```diff
@@ -12,36 +12,14 @@
 from ansys.fluent.core.codegen.data.meshing_utilities_examples import (
     meshing_utility_examples,
 )
+from ansys.fluent.core.constants import _PY_TYPE_BY_DM_TYPE
 from ansys.fluent.core.utils.fluent_version import (
     FluentVersion,
     get_version_for_file_name,
 )
 
 _ROOT_DIR = Path(__file__) / ".." / ".." / ".." / ".." / ".." / ".."
 
-_PY_TYPE_BY_DM_TYPE = {
```

Comment:
@hpohekar Similar to my other comment in `flunits`, I cannot see the benefit of this. cc: @prmukherj

Reply:
@seanpearsonuk, if you see the task item: https://github.com/ansys/pyfluent/issues/3405, created by Mainak, this will solve a lot of the circular dependency issues. I think that's the only benefit.
cc. @hpohekar

Reply:
@prmukherj The benefit associated with the proposal in #3405 is not provided by this change. These are private module constants. Would you extract all private module constants from and put them all in one module? Absolutely never. Lets not code like robots.